go/vt/schemadiff/table.go

```diff
@@ -1234,13 +1266,15 @@ func (c *CreateTableEntity) diffPartitions(alterTable *sqlparser.AlterTable,
 	case t1Partitions == nil:
 		// add partitioning
 		alterTable.PartitionOption = t2Partitions
+		annotations.MarkAdded(sqlparser.CanonicalString(t2Partitions))
```

Comment:
Any reason not to add a test case for this in the unit test?

Reply:
Added.